packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -298,6 +298,32 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
         return queryTypes;
     }
 
+    protected createCustomRootTypes(inputRootType: GraphQLObjectType) {
```

Comment:
What this function does?

Reply:
This sounds like something we should not do. 
What I see that this code will strip out all directives from types. 
Is that desired?

Reply:
Generally, we can go back and use mergeType from toolkit - writing our own helper in plugin is risky business and then this also will mean complexity for other plugins doing schema

Reply:
Actually this is a bad name now that I think about it. This function simply receives an object input and maps the fields.

So it will receive a query object, and maps the fields into a field object in the same format that is used to create the CRUD objects (see https://github.com/aerogear/graphback/blob/124a3873e2300353bd239cc9de1a9d2eac419eb4/packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts#L282)